src/Framework/Framework/Controls/KnockoutBindingGroup.cs

```diff
@@ -22,22 +22,38 @@ public virtual void Add(string name, DotvvmControl control, DotvvmProperty prope
             if (binding == null)
             {
                 if (nullBindingAction != null) nullBindingAction();
-                else Add(name, JsonConvert.SerializeObject(control.GetValue(property), DefaultSerializerSettingsProvider.Instance.Settings));
+                else AddValue(name, control.GetValue(property));
             }
             else
             {
                 entries.Add(new KnockoutBindingInfo(name, GetKnockoutBindingExpression(control, binding)));
             }
         }
 
-        public virtual void Add(string name, string expression, bool surroundWithDoubleQuotes = false)
+        public virtual void Add(string name, string expression)
+        {
+            entries.Add(new KnockoutBindingInfo(name, expression));
+        }
+
+        public void Add(string name, KnockoutBindingGroup nestedGroup)
+        {
+            if (!nestedGroup.IsEmpty)
+                Add(name, nestedGroup.ToString());
```

Comment:
Add support for combining knockout groups

Reply:
We will solve this in another PR